### DISCUSSION TITLE
Web app?

### DISCUSSION THREAD

**Discussion by FlutterFraz** [Ivy-Apps/ivy-wallet#2944] [Ideas]
Hello, 
I hope this question won't result too much naive.

I really like to jump from my phone to my workstation and I was wondering if there was any possibility of running a web app on a LXC to be synced with my Android app.

**Comment by ILIYANGERMANOV** [comment]
Hey @FlutterFraz a web app is a good idea but we have a few problems:
- We have no BE/cloud to sync the data
- We can do a web app using Kotlin and Compose Multiplatform but have to wait for Room and Hilt to support KMP first

**Reply by FlutterFraz** [reply]
About syncing the data, I was thinking about the web app since I have already a domain configured with nginx and I totally bypass the cloud problem.

The totally uncanny way I was thinking about was to somehow build an Android container without any access to the web except for my sync service which would pull data from the ivy folder on my phone but I don't even think it's possible to achieve that

**Reply by ILIYANGERMANOV** [reply]
Hey @FlutterFraz there's a community project that attempts to build an Ivy Wallet web app. Dropping it here if you want to check it out:

https://github.com/pratikkabade/ivy-wallet-web
